.github/workflows/cloud-integration-tests.yml

```diff
@@ -17,10 +17,6 @@ on:
           with the prefix "test_index". Even those that were not created by this workflow.'
         required: true
         default: 'run_integration_tests'
-  pull_request:
-    branches:
-      - mainline
-      - 'releases/*'
```

Comment:
Why don't we run this on every push to PR? How do we make sure the cloud integration tests is run before the release?